### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by ncateuc** [share121/inter-knot#845] [其他]
我这个情况怎么回事？各位知道怎么处理吗？
![Screenshot_2024-08-10-17-39-04-43_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/642cda77-7e0f-4e14-8d07-7ce9923a18c7)

**Comment by ncateuc** [comment]
![Screenshot_2024-08-10-17-39-04-43_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/5bc59bdb-0b21-4fbe-af4f-3b99e908fcbe)

**Comment by NOW363** [comment]
右下角开电脑模式
